components/Header.tsx

```diff
@@ -11,14 +12,15 @@ function Header() {
                TipoHiz.__
             </p>
          </div>
-         <div className='flex justify-between items-center gap-3 font-semibold'>
+         <div className="flex justify-between items-center gap-3 font-semibold">
             <div className="flex items-center justify-center p-2 rounded-xl gap-3 cursor-pointer text-foreground bg-input">
                <p className="text-sm tracking-wide text-background">Login</p>
-               <LuUser className='text-background' />
+               <LuUser className="text-background" />
```

Comment:
I don't neither your husky nor prettier works.

We use single quotes. check .prettierrc